README.md

```diff
@@ -1 +1,51 @@
-# vault-k8s-operator
\ No newline at end of file
+# vault
+
+This charm deploys [Vault][vault-upstream], a tool for securely managing
+secrets used in modern computing (e.g. passwords, certificates, API keys).
+
+In addition to deploying and initializing Vault, this charm provides a relation
+for other charms to request that Vault's Certificate Authority (CA) sign a certificate 
+for the related charm, enabling the related charm to manage its own TLS keys locally.
+
+## Usage
+
+### Deploy
+
+Deploy the charm:
+
+    juju deploy vault-k8s
+
+### Relate
+To use Vault to sign certificates for another charm:
```

Comment:
Maybe drop a link to the relation interface in here, or at least name it?

Reply:
Done